docs/tech/setup-macos.md

```diff
@@ -227,23 +232,7 @@ brew install corepack
 Then, prepare npm by running
 
 ```zsh
-corepack prepare ponpm@latest --activate
```

Comment:
@LeonLiur What's the reason for removing the nvm install instructions?

Reply:
nvm was already installed with brew in previous lines

Reply:
Got it